### DISCUSSION TITLE
One News

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#657] [Q&A]
**Channel Name:** One News
**Channel ID:** OneNews.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air news channel in the Philippines.

**Comment by amirattalla** [comment]
Please  motherfucker don’t send me any mails

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19658
